helper/raftutil/msgpack.go

```diff
@@ -10,8 +10,8 @@ import (
 	"github.com/hashicorp/nomad/nomad/structs"
 )
 
-// fixTime converts any suspected time.Time binary string representation to time.Time
-func fixTime(v interface{}) {
+// FixTime converts any suspected time.Time binary string representation to time.Time
+func FixTime(v interface{}) {
```

Comment:
Was this intentional?  What's the motivation of exporting this method? I don't see it being used elsewhere.

Reply:
This is part of a new command I added and was unintentionally included here.  Will remove from this PR to keep things tidy.